v5analytics/visallo
# Product fixes

Some product improvements
* Better fit when dragging into product list
* Show unknown concepts in map
* Allow file upload in map, dashboard (won't add to product, but will
select it) Graph works same as before – it will add

Testing Instructions: 
Drop file (for upload) works in map and dashboard (although it doesn't add to map like graph)
Dragging entity into product list for map should open the map
Entities with an unknown concept will show in map

CHANGELOG
Changed: Map and dashboard can accept file uploads
Fixed: Dropping elements in map products in the product list will open the map
